src/parser.ts

```diff
@@ -37,6 +37,20 @@ export function parse(
   usedNames = new Set<string>()
 ): AST {
   if (isPrimitive(schema)) {
+    if (options.booleanSchemas) {
+      if (schema === true) {
+        return {
+          keyName,
+          type: 'ANY'
```

Comment:
Careful to respect the `unknownAny` option here. Could you make sure your tests cover what happens when the option is set to `true`/`false`?
```suggestion
          type: options.unknownAny ? 'UNKNOWN' : 'ANY'
```